src/main/java/net/starlark/java/eval/StarlarkInt.java

```diff
@@ -403,6 +422,12 @@ public static int compare(StarlarkInt x, StarlarkInt y) {
     if (x instanceof Int32 && y instanceof Int32) {
       return Integer.compare(((Int32) x).v, ((Int32) y).v);
     }
+
+    try {
+      return Long.compare(x.toLongFast(), y.toLongFast());
+    } catch (OverflowException ignore) {
```

Comment:
Call it 'unused', as this suppresses an annoying static check that runs on Google's Java code.

Reply:
Renamed to `unused`, but note that Idea shows warning about `unused` parameter, but doesn't for `ignorre`.

Reply:
Thanks. I've reported a bug against our static checker to request that it agree with Idea.